### PR TITLE
Include instructions on how to find a device's MAC address

### PR DESCRIPTION
**Pull request by polarapfel**
Quick pointer to how to retrieve the device's MAC address.

### CODE REVIEW

File: docs/_docs/03-faq.md [hunk 1]
```diff
@@ -84,6 +84,10 @@ To resolve, try to use Wi-Fi with normal WPA/WPA2 authentication.
 
 Currently, DevKit only can connect to 2.4 GHz Wi-Fi, 5 GHz is not supported due to hardware restrictions.
 
+### The WiFi network I am connecting to is restricted and I need to register the deviceS MAC address. Where can I find it?
+
```
Comment (ArthurMa1978) [inline docs/_docs/03-faq.md]: Thank you for providing this instructions.
Actually the MAC address is printed on the metal shell as bellow, would you mind also add this in this FAQ?
image

Comment (lirenhe) [reply docs/_docs/03-faq.md]: I would suggest entering into 'configuration mode', hold button A and click reset button, the mac address is shown with id:XXXXX

Comment (polarapfel) [reply docs/_docs/03-faq.md]: Both good points. I wasn't aware of either.

Comment (ArthurMa1978) [reply docs/_docs/03-faq.md]: Thank you @polarapfel for pointing this out what we were missing in our documents.
I'd like to close this pr and add a document into our how-to docset.

**Comment by ArthurMa1978** [comment]
https://github.com/Microsoft/azure-iot-developer-kit/issues/293 was created to track this work.
Close this pr.

**Pull request closed without merging**